pyanaconda/modules/payloads/source/source_base.py

```diff
@@ -58,6 +58,14 @@ def description(self):
         """
         pass
 
+    @property
+    def network_required(self):
+        """Does the source require a network?
+
+        :return: True or False
+        """
+        return False
```

Comment:
Maybe this should be abstract like the rest of these properties? Not sure.

Reply:
Fixed.